lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -1,130 +1,174 @@
   state:
     description:
-      - The database extension state
+    - The database extension state.
     default: present
-    choices: [ "present", "absent" ]
+    choices: [ absent, present ]
+    type: str
   cascade:
     description:
-      - Automatically install/remove any extensions that this extension depends on
-        that are not already installed/removed (supported since PostgreSQL 9.6).
+    - Automatically install/remove any extensions that this extension depends on
+      that are not already installed/removed (supported since PostgreSQL 9.6).
     type: bool
     default: no
     version_added: '2.8'
 notes:
-   - The default authentication assumes that you are either logging in as or sudo'ing to the C(postgres) account on the host.
-   - This module uses I(psycopg2), a Python PostgreSQL database adapter. You must ensure that psycopg2 is installed on
-     the host before using this module. If the remote host is the PostgreSQL server (which is the default case), then PostgreSQL must also be installed
-     on the remote host. For Ubuntu-based systems, install the C(postgresql), C(libpq-dev), and C(python-psycopg2) packages on the remote host before using
-     this module.
+- The default authentication assumes that you are either logging in as
+  or sudo'ing to the C(postgres) account on the host.
+- This module uses I(psycopg2), a Python PostgreSQL database adapter.
+- You must ensure that psycopg2 is installed on the host before using this module.
+- If the remote host is the PostgreSQL server (which is the default case),
+  then PostgreSQL must also be installed on the remote host.
+- For Ubuntu-based systems, install the C(postgresql), C(libpq-dev),
+  and C(python-psycopg2) packages on the remote host before using this module.
 requirements: [ psycopg2 ]
 author:
-    - "Daniel Schep (@dschep)"
-    - "Thomas O'Donnell (@andytom)"
+- Daniel Schep (@dschep)
+- Thomas O'Donnell (@andytom)
 '''
 
-EXAMPLES = '''
-# Adds postgis to the database "acme"
-- postgresql_ext:
+EXAMPLES = r'''
+- name: Adds postgis extension to the database acme in the schema foo
+  postgresql_ext:
     name: postgis
     db: acme
-    schema: extensions
+    schema: foo
 
-# Adds earthdistance to the database "template1"
-- postgresql_ext:
+- name: Removes postgis extension to the database acme
+  postgresql_ext:
+    name: postgis
+    db: acme
+    state: absent
+
+- name: Adds earthdistance extension to the database template1 cascade
+  postgresql_ext:
     name: earthdistance
     db: template1
     cascade: true
+
+# In the example below, if earthdistance extension is installed,
+# it will be removed too because it depends on cube:
+- name: Removes cube extension from the database acme cascade
+  postgresql_ext:
+    name: cube
+    db: acme
+    cascade: yes
+    state: absent
+'''
+
+RETURN = r'''
+query:
+  description: List of executed queries.
+  returned: always
+  type: list
+  sample: ["DROP EXTENSION \"acme\""]
```

Comment:
A slightly cleaner version that avoids escaping:
```suggestion
  sample: ['DROP EXTENSION "acme"']
```

Reply:
fixed, thank you